precommit/src/main/shell/plugins.d/shellcheck.sh

```diff
@@ -62,6 +62,7 @@ function shellcheck_precheck
   if ! verify_command "shellcheck" "${SHELLCHECK}"; then
     add_vote_table_v2 0 shellcheck "" "Shellcheck was not available."
     delete_test shellcheck
+    return 0
```

Comment:
None of the other branches below need an explicit return value?

Reply:
Not really.  I made this return explicit to match the other prechecks. Being explicit, it re-enforces that test-patch really shouldn't fail if a particular tool isn't around and we definitely don't want the rest of the code to continue since that has a few side-effects.